CMakeLists.txt

```diff
@@ -507,6 +507,10 @@ if (BUILD_TESTS OR BUILD_BENCHMARKS)
     set(HAVE_CMAKE_GTEST OFF)
   endif()
 
+  # Some benchmarks require an input sample file.
+  file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/folly/test/input.json
+     DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
CONTENT_DIR exists and is what's intended to do this. `input.json` will need to be moved into it's own directory though.